lib/Conf.ml

```diff
@@ -1660,7 +1663,7 @@ let janestreet_profile =
   ; disable= false
   ; disambiguate_non_breaking_match= false
   ; doc_comments= `Before
-  ; doc_comments_val= `Before
+  ; doc_comments_val= `Unset
```

Comment:
This is the `janestreet` profile, I don't expect this to cause regressions.

Reply:
I didn't see regressions with the doc-comments on core_kernel, so it looks safe to me